instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -1700,14 +1770,23 @@ def test_invoke_model_no_content_different_events(
     assert_message_in_logs(logs[4], "gen_ai.choice", choice_body, span)
 
 
+@pytest.mark.parametrize(
+    "model_family",
+    ["amazon.nova", "anthropic.claude"],
+)
 @pytest.mark.vcr()
 def test_invoke_model_no_content_tool_call(
     span_exporter,
     log_exporter,
     bedrock_runtime_client,
     instrument_no_content,
+    model_family,
 ):
-    llm_model_value = "us.anthropic.claude-3-5-sonnet-20240620-v1:0"
+    if model_family == "amazon.nova":
```

Comment:
There's a lot of branhcing in this test, I wonder if splitting it into two tests would be clearer than parameterizing

Reply:
I'll try to split the test helper in one function per model family

Reply:
I've moved the per-model part into some classes that get passed to the test helper depending on the model so that there are no more model related branches in the test helper.